skyportal/tests/fixtures.py

```diff
@@ -107,6 +107,7 @@ class Meta(BaseMeta):
     mjd = factory.LazyFunction(lambda: 58000.0 + np.random.random())
     flux = factory.LazyFunction(lambda: 20 + 10 * np.random.random())
     fluxerr = factory.LazyFunction(lambda: 2 * np.random.random())
+    owner_id = 1
```

Comment:
= Provisioned admin

Reply:
Can you add that as a comment?  Is this guaranteed?